src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/validation/EnvelopeProcessorValidationTest.java

```diff
@@ -266,6 +268,22 @@ public void should_throw_an_exception_when_jurisdiction_poBox_and_container_dont
         verifyExceptionIsThrown(envelope, container, err);
     }
 
+    @Test
+    public void should_throw_an_exception_when_zip_filename_does_not_match_with_metadata() {
+        // given
+        InputEnvelope envelope = inputEnvelope("test_jurisdiction"); // metadata zip file name "file.zip"
+
+        // when
+        Throwable throwable = catchThrowable(
+            () -> assertZipFilenameMatchesWithMetadata(envelope, "invalid-zip-filename.zip")
+        );
+
+        // then
+        assertThat(throwable)
+            .isInstanceOf(ZipNameNotMatchMetaDataException.class)
+            .hasMessage("Zip file name uploaded does not match with field \"zip_file_name\" in the metadata");
```

Comment:
how about "Name of uploaded zip file...."?

Reply:
updated the error message